src/SFML/Audio/Miniaudio.cpp

```diff
@@ -29,5 +29,5 @@
 #pragma GCC diagnostic ignored "-Wnull-dereference" // Work around MinGW warning
 #endif
 // #define MA_DEBUG_OUTPUT // Uncomment to enable miniaudio debug output to console
-#define MINIAUDIO_IMPLEMENTATION
-#include <miniaudio.h>
+// NOLINTNEXTLINE(bugprone-suspicious-include)
```

Comment:
We can more simply just put a `// NOLINT(check name here)` on the same line

Reply:
Done.